storages/backends/gcloud.py

```diff
@@ -34,6 +37,7 @@
 
 CONTENT_ENCODING = "content_encoding"
 CONTENT_TYPE = "content_type"
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Please remove the logger definition & import if it's unused.

Reply:
done.